workbench-core/accounts/src/services/projectService.ts

```diff
@@ -186,32 +201,22 @@ export default class ProjectService {
     // Generate Project ID
     const projectId = uuidWithLowercasePrefix(resourceTypeToKey.project);
 
-    // Create ProjectAdmin and Researcher groups--TODO implement after dynamic AuthZ
-    // const createProjectAdminGroupResponse = this._dynamicAuthorizationService.createGroup(user, {
-    //   groupId: `${projId}#PA`,
-    //   description: `Project Admin group for ${projId}`
-    // });
-    // const createResearcherGroupResponse = this._dynamicAuthorizationService.createGroup(user, {
-    //   groupId: `${projId}#Researcher`,
-    //   description: `Researcher group for ${projId}`
-    // });
-    // if (!createProjectAdminGroupResponse.created || !createResearcherGroupResponse.created) {
-    //   throw Boom.badImplementation(
-    //     'Failed to create Project Admin group or Researcher group with dynamic authorization service.'
-    //   );
-    // }
-
-    // Create Permissions for the groups--TODO implement after dynamic AuthZ
-    // const identityPermissions: IdentityPermission[] = this._generateIdentityPermissionsForProject(projectId);
-    // const createIdentityPermissionsResponse = this._dynamicAuthorizationService.createIdentityPermissions(
-    //   user,
-    //   identityPermissions
-    // );
-    // if (!createIdentityPermissionsResponse.created) {
-    //   throw Boom.badImplementation(
-    //     'Failed to create batch identity permissions for project with dyamic authorization service.'
-    //   );
-    // }
+    await this._dynamicAuthorizationService.createGroup({
```

Comment:
Why don't we save and check these groups were successfully created?

Reply:
DynamicAuthService throws if the call isn't successful

Reply:
i'm good with that. I thought they returned a boolean for it.